assets/js/aboutCodeDataTables.js

```diff
@@ -159,6 +159,16 @@ class AboutCodeDataTable {
             "ajax": (dataTablesInput, dataTablesCallback) =>
                 this._query(dataTablesInput, dataTablesCallback),
             "columns": AboutCodeDataTable.TABLE_COLUMNS,
+            "columnDefs": [
+                {
+                    "targets": [ 2, 3, 4, 5, 6, 7, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 25, 26, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47 ],
```

Comment:
@johnmhoran Is there any way you could list the targets by name instead of index? Because the index isn't very clear on what is hidden/visible and will break if the columns defined in ABCDT are reordered. If it does not allow referencing by name, then can you put the visibility directly on the column definitions in ABCDT?